packages/changeset-validations/src/helpers/validate-changeset.ts

```diff
@@ -0,0 +1,51 @@
+import { isChangeset } from 'validated-changeset';
+
+import type { ErrorRecord, FormValidateCallback } from 'ember-headless-form';
+import type { EmberChangeset } from 'ember-changeset';
+import { assert } from '@ember/debug';
+
+const validateChangeset: FormValidateCallback<EmberChangeset> = async (
```

Comment:
Suggestion: Can we get some JSDOC here with a code sample?

Pseudocode, not tested...
```
import validateChangeset from '@ember-headless-form/changeset/helpers/validate-changeset';

const changeset = Changeset(data, validator);

// in template
 <HeadlessForm
        @data={{changeset}}
        @dataMode="mutable"
        @validate={{validateChangeset}}
        @onInvalid={{invalidHandler}}
        as |form|
      >
      ```

Reply:
Good point! Added this ticket: https://github.com/CrowdStrike/ember-headless-form/issues/38